examples/encoding/plot_continuous_encoder.py

```diff
@@ -0,0 +1,114 @@
+"""
+=============================
+Regression on continuous data
+=============================
+
+This demonstrates how an encoding model can be fit with multiple continuous
+inputs. In this case, a encoding model is fit for one EEG channel, using the
+values of all other electrodes as inputs. The coefficients in this case are
+interpreted as a measure of functional connectivity. These inputs could also
+be stimulus features, such as spectral features of sound. In this case, model
+coefficients would be measures of receptive field properties for the channel.
+
+Estimation of spectro-temporal and spatio-temporal receptive fields using
+modeling with continuous inputs is described in:
+
+    Theunissen, F. E. et al. Estimating spatio-temporal receptive
+    fields of auditory and visual neurons from their responses to
+    natural stimuli. Network 12, 289-316 (2001).
+
+    Willmore, B. & Smyth, D. Methods for first-order kernel
+    estimation: simple-cell receptive fields from responses to
+    natural scenes. Network 14, 553-77 (2003).
+"""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#
+# License: BSD (3-clause)
+
+import matplotlib.pyplot as plt
+
+import mne
+import numpy as np
+from mne.datasets import sample
+from mne.encoding import SampleMasker, get_coefs, get_final_est, FeatureDelayer
+from sklearn.pipeline import make_pipeline
+from sklearn.linear_model import Ridge
+from sklearn.preprocessing import StandardScaler
+
+###############################################################################
+# Load and preprocess data
+data_path = sample.data_path()
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+raw = mne.io.read_raw_fif(raw_fname, preload=True).pick_types(
+    meg=False, stim=False, eeg=True).filter(1, None, method='iir')
+seed_elec = 'EEG 011'
+ix_seed = mne.pick_channels(raw.ch_names, [seed_elec])[0]
+ix_targets = np.setdiff1d(range(len(raw.ch_names)), [ix_seed])
+fig = raw.plot_sensors()
+colors = np.zeros(len(raw.ch_names)).astype(bool)
+colors[ix_seed] = True
+colors = np.where(colors, 'r', 'k')
+plt.setp(fig.axes[0].collections[0], facecolors=colors, edgecolors=colors)
+fig.suptitle('Sensor positions (seed = red)')
+
+# Set up encoding model variables
+X = raw[:][0][ix_targets]
+y = raw[:][0][ix_seed]
+n_targets, n_times = X.shape
+
+###############################################################################
+# Fit the model / Make predictions
+# Define model preprocessing and parameters
+ix_split = int(n_times * .998)  # Using subset of data so it fits in a plot
```

Comment:
I was using .998 just to make the plot understandable, otherwise you plot many seconds of EEG data and it's impossible to compare predictions and actual values. I could just use a 50/50 split but only plot a subset of the predictions instead.